script/releases/v0.1.0-rewardsv2/1-eoa.s.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import {RewardsCoordinator} from "src/contracts/core/RewardsCoordinator.sol";
+import {IDelegationManager} from "src/contracts/interfaces/IDelegationManager.sol";
+import {DelegationManager} from "src/contracts/core/DelegationManager.sol";
+import {StrategyManager} from "src/contracts/core/StrategyManager.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+import {Test, console} from "forge-std/Test.sol";
+
+contract Deploy is EOADeployer {
+    using EigenLabsUpgrade for *;
+
+    function _deploy() internal override {
+        vm.startBroadcast();
+        
+        deploySingleton(
+            address(
+                new RewardsCoordinator(
+                    IDelegationManager(zDeployedProxy(type(DelegationManager).name)),
+                    StrategyManager(zDeployedProxy(type(StrategyManager).name)),
+                    zUint32("REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS"),
+                    zUint32("REWARDS_COORDINATOR_MAX_REWARDS_DURATION"),
+                    zUint32("REWARDS_COORDINATOR_MAX_RETROACTIVE_LENGTH"),
+                    zUint32("REWARDS_COORDINATOR_MAX_FUTURE_LENGTH"),
+                    zUint32("REWARDS_COORDINATOR_GENESIS_REWARDS_TIMESTAMP")
+                )
+            ),
+            this.impl(type(RewardsCoordinator).name)
+        );
+
+        vm.stopBroadcast();
+    }
+
+    function testDeploy() public {
+        _deploy();
+        Deployment[] memory deploys = deploys();
+        console.log(zDeployedImpl(type(RewardsCoordinator).name));
+
+        RewardsCoordinator rewards = RewardsCoordinator(zDeployedImpl(type(RewardsCoordinator).name));
```

Comment:
- @wadealexc @nadir-akhtar @0xrajath  - zeus tests are now first class `Test`, and can contain `test*` methods.
- i filter them out in our foundry.toml by default, as they should only ever be run with `zeus test --env <env> [script]`